detection-rules/attachment_html_smuggling_atob.yml

```diff
@@ -0,0 +1,24 @@
+name: "Attachment: HTML smuggling with atob and high entropy"
+description: |
+  Recursively scans files and archives to detect HTML smuggling techniques.
+type: "rule"
+source: |
+  type.inbound
+  and any(attachments, .size <= 15000 and
```

Comment:
Is this `.size` supposed to apply to all the top-level files or the exploded files?